DOCSP-27445: instantiate collection in insert examples (#517)
mongodb/docs-node: source/fundamentals/crud/write-operations/insert.txt

Example
~~~~~~~

The following example creates and inserts a document to the

> ``myDB.pizzaMenu`` collection using the ``insertOne()`` method:

Author: reworded

.. code-block:: javascript

>    const myDB = client.db("myDB")

Reviewer: ```suggestion
   const myDB = client.db("myDB");
```

   .. code-block:: javascript

>       const myDB = client.db("myDB")

Reviewer: ```suggestion
      const myDB = client.db("myDB");
```

Example
~~~~~~~

> The following example creates and inserts three documents to the
> ``myDB.pizzaMenu`` collection using the ``insertMany()`` method:

Reviewer: I: [Style Guide entry on "using"](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#u)

```suggestion
The following example creates three documents and uses the
``insertMany()`` method to insert them into the
``myDB.pizzaMenu`` collection:
```
